src/main/features/dashboard/views/claim-details.njk

```diff
@@ -0,0 +1,41 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import linkButton, csrfProtection %}
+
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">
+        <span class="form-hint">{{ claimNumber }}</span>
+        {{ claimantVDefendant }}
+      </h1>
+      <h3 class="heading-medium">{{ t('Date Issued') }}
+        <div class="form-label">{{ dateIssued | date }}</div>
+      </h3>
+
+      <h3 class="heading-medium">{{ t('Claimant(s)') }}
```

Comment:
Why don't we display `"(s"` only when claimants.length > 1?

Reply:
done